plasmapy/atomic/atomic.py

```diff
@@ -180,186 +176,13 @@ def standard_atomic_weight(element: Particle) -> u.Quantity:
-    weight of an element or the atomic mass of an isotope, and then
-    accounting for the change in mass due to loss of electrons from
-    ionization.
-
-    Parameters
-    ----------
-    particle: `str`, `int`, or `~plasmapy.atomic.Particle`
-        A string representing an element, isotope, or ion; an integer
-        representing an atomic number; or an instance of the Particle
-        class.
-
-    Z: `int`, optional
-        The ionization state of the ion (defaulting to a charge of
-        `Z = 1`)
-
-    mass_numb: `int`, optional
-        The mass number of an isotope.
-
-    Returns
-    -------
-    m_i: `~astropy.units.Quantity`
-        The mass of a single ion of the isotope or element with charge
-        state `Z`.
-
-    Raises
-    ------
-    `TypeError`
-        The argument is not a `str`, `int`, or
-        `plasmapy.utils.~Particle`.
-
-    `~plasmapy.utils.InvalidParticleError`
-        If the arguments do not correspond to a valid particle.
-
-    `~plasmapy.utils.InvalidIonError`
-        If the argument represents a particle other than an ion, the
-        ionization state exceeds the atomic number, or no isotope mass
-        or standard atomic weight is available.
-
-    `~plasmapy.utils.MissingAtomicDataError`
-        If the standard atomic weight or isotopic mass is not known.
-
-    See Also
-    --------
-    `~plasmapy.atomic.standard_atomic_weight` : returns the conventional
-        atomic mass of an element based on terrestrial values and
-        assuming the atom is neutral.
-
-    `~plasmapy.atomic.isotope_mass` : returns the mass of an isotope (if
-        available) assuming the atom is neutral.
-
-    Notes
-    -----
-    This function in general finds the mass of an isotope (or the
-    standard atomic weight based on terrestrial values if a unique
-    isotope cannot be identified), and then subtracts the mass of `Z`
-    electrons.
-
-    If `Z` is not provided as an input, then `ion_mass` assumes that the
-    ion is singly ionized while issuing a `DeprecationWarning`.
-
-    Specific values are returned for some special particles such as
-    protons, deuterons, tritons, and positrons.
-
-    Calling `ion_mass('H')` does not return the mass of a proton but
-    instead uses hydrogen's standard atomic weight based on
-    terrestrial values.  To get the mass of a proton, use
-    `ion_mass('p')`.
-
-    Examples
-    --------
-    >>> ion_mass('H+')  # assumes terrestrial abundance of D
-    <Quantity 1.67291241e-27 kg>
-    >>> ion_mass('H+') == ion_mass('p')
-    False
-    >>> ion_mass('P+')  # phosphorus
-    <Quantity 5.14322301e-26 kg>
-    >>> ion_mass('He-4', Z=2)
-    <Quantity 6.64465709e-27 kg>
-    >>> ion_mass('T+')
-    <Quantity 5.00735666e-27 kg>
-    >>> ion_mass(26, Z=1, mass_numb=56)
-    <Quantity 9.28812345e-26 kg>
-    >>> ion_mass('Fe-56 1+')
-    <Quantity 9.28812345e-26 kg>
-
-    """
-
-    # TODO: Remove deprecated functionality elsewhere in the code
```

Comment:
So this is the PR that removes all the `DeprecationWarning`s? Awesome!

Reply:
Some of them, at least!  There are still a bunch that are popping up for other tests.